swingset-demo/src/main/java/com/nqadmin/swingset/demo/DataGridExampleSupport.java

```diff
@@ -183,7 +183,7 @@ private void setAllowDelete(AbstractButton b) {
 				+ "</center></html>");
 	}
 
-	@SuppressWarnings("deprecation")
+	@SuppressWarnings({"deprecation", "UseOfSystemOutOrSystemErr"})
```

Comment:
Do we want to suppress deprecation warnings or should we try to fix them? Maybe no good alternative?